### ISSUE TITLE
Unable to update anything

### ISSUE THREAD

**Issue by lemieuxl** [conda/conda#3468]
When trying to update conda or all packages on Windows (after `conda` was updated to 4.2.7), I get the following error:

```
An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues



Current conda install:

               platform : win-64
          conda version : 4.2.7
       conda is private : False
      conda-env version : 4.2.7
    conda-build version : not installed
         python version : 3.4.4.final.0
       requests version : 2.11.1
       root environment : C:\Users\lemieuxl\Miniconda3  (writable)
    default environment : C:\Users\lemieuxl\Miniconda3
       envs directories : C:\Users\lemieuxl\Miniconda3\envs
          package cache : C:\Users\lemieuxl\Miniconda3\pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                          https://repo.continuum.io/pkgs/free/noarch/
                          https://repo.continuum.io/pkgs/pro/win-64/
                          https://repo.continuum.io/pkgs/pro/noarch/
                          https://repo.continuum.io/pkgs/msys2/win-64/
                          https://repo.continuum.io/pkgs/msys2/noarch/
            config file : C:\Users\lemieuxl\Miniconda3\.condarc
           offline mode : False



`$ C:\Users\lemieuxl\Miniconda3\Scripts\conda-script.py update conda`




    Traceback (most recent call last):
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\exceptions.py", line 472, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\cli\main.py", line 144, in _main
        exit_code = args.func(args, p)
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\cli\main_update.py", line 65, in execute
        install(args, parser, 'update')
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\cli\install.py", line 308, in install
        update_deps=context.update_dependencies)
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\plan.py", line 526, in install_actions
        force=force, always_copy=always_copy)
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\plan.py", line 308, in ensure_linked_actions
        fetched_in = is_fetched(dist)
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\install.py", line 723, in is_fetched
        for fn in package_cache().get(dist, {}).get('files', ()):
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\install.py", line 671, in package_cache
        add_cached_package(pdir, url)
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\install.py", line 629, in add_cached_package
        schannel = Channel(url).canonical_name
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\models\channel.py", line 50, in __call__
        self.__init__(value)
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\models\channel.py", line 144, in __init__
        if not has_scheme(url):
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\models\channel.py", line 28, in has_scheme
        return bool(urlparse(value).scheme in RECOGNIZED_URL_SCHEMES)
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\conda\common\url.py", line 51, in urlparse
        return parse_url(url)
      File "C:\Users\lemieuxl\Miniconda3\lib\site-packages\requests\packages\urllib3\util\url.py", line 189, in parse_url
        raise LocationParseError(url)
    requests.packages.urllib3.exceptions.LocationParseError: Failed to parse: E:\miniconda_channel
```

The local channel `file://E:\miniconda_channel` was removed, but the error persists (as shown above).

**Comment by JeremyEnns** [comment]
It looks like the windows file location parsing has changed.  I was able to get it to work using this method.

`file:///C:/custom/pkgs` Note the 3 initial slashes.

**Comment by RutgerK** [comment]
@JeremyEnns Thanks for the tip, but how have you replaced all references? I have a dozen environments with a lot of packages from local repo's. 

The 'global' references are easy to replace, the channels in the .condac, and the urls.txt (pkgs dir) files. 

Have you also changed every json file in the conda-meta directory which is located within each environment? That would be a lot of work, or can you safely remove those json's without breaking the environment?

**Comment by kalefranz** [comment]
This should be resolved with #3550.
